examples/megacli.cpp

```diff
@@ -3316,6 +3316,40 @@ void exec_cp(autocomplete::ACState& s)
 
 void exec_du(autocomplete::ACState& s)
 {
+    if (s.extractflag("-all"))
+    {
+        bool breakdown = s.extractflag("-breakdown");
+        struct counters { m_off_t filecount = 0, filebytes = 0, fileversioncount = 0, fileversionbytes = 0, foldercount = 0; };
```

Comment:
structs are pascal case

Reply:
surely for a local struct, with hardly any scope, in a non customer facing app, it doesn't matter?

Reply:
All classes in the SDK are pascal case no? That's also what we agreed on when we discussed naming on slack.

Reply:
I agree with Christian here. We should get used to the code style conventions we agreed, even if the code belongs to an example, a test or the main core of the library :)